setuptool/setuptool.go

```diff
@@ -32,8 +32,17 @@ func main() {
 	flag.Parse()
 	fmt.Printf("Welcome to the Web Api Gateway Config Tool.\n\n")
 
+	c, _, err := config.ReadWriteConfig()
+	if err != nil {
+		fmt.Printf("Unable to load config file %v\n", err)
```

Comment:
should you return and not continue if there is error?

Reply:
Thanks! This should never happen, so i would just go with os.Exit(1）